src/lib/support/DefaultStorageKeyAllocator.h

```diff
@@ -99,7 +99,7 @@ class DefaultStorageKeyAllocator
     static StorageKeyName FabricNOC(FabricIndex fabric) { return StorageKeyName::Formatted("f/%x/n", fabric); }
     static StorageKeyName FabricICAC(FabricIndex fabric) { return StorageKeyName::Formatted("f/%x/i", fabric); }
     static StorageKeyName FabricRCAC(FabricIndex fabric) { return StorageKeyName::Formatted("f/%x/r", fabric); }
-    static StorageKeyName FabricVVSC(FabricIndex fabric) { return StorageKeyName::Formatted("f/%x/vvvc", fabric); }
+    static StorageKeyName FabricVVSC(FabricIndex fabric) { return StorageKeyName::Formatted("f/%x/vvsc", fabric); }
```

Comment:
I assume we claim "not yet released" so no backwards compatibility issues?

Reply:
Yes. Unless you ran TC-OPCREDS-3.8 (which almost for sure nobody did), you would not have any data there.